packages/test/test-version-utils/src/versionUtils.ts

```diff
@@ -251,7 +251,9 @@ export async function ensureInstalled(
 						if (error) {
 							reject(
 								new Error(
-									`Failed to initialize install directory ${modulePath}\n${stderr}`,
+									`Failed to initialize install directory ${modulePath}\nError:${JSON.stringify(
```

Comment:
Is error actually an `Error`? Do you want the stack?

Reply:
Good point, updated